website/management/commands/fetch_github_comments.py

```
@@ -0,0 +1,263 @@
+
+                # Process each comment
+                for comment in comments:
+                    self.save_comment(issue, comment)
+
+                # Check if there are more pages
+                if len(comments) < 100:
+                    break
+
+                page += 1
+
+        except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)